hassil/__init__.py

```
@@ -1,11 +1,13 @@
 """Home Assistant Intent Language parser"""
 
 from .expression import (
+    Alternative,
```

Ruff (0.8.2)

`.expression.Alternative` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)